assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -64,9 +59,14 @@ export const TakeActionBoxoutEditor = ({
 
     // eslint-disable-next-line no-shadow
     const actPageList = [].concat(
-      select('core').getEntityRecords('postType', 'page', args) || [],
-      select('core').getEntityRecords('postType', 'p4_action', actionsArgs) || []
-    ).sort((a, b) => {
+      select('core').getEntityRecords('postType', 'page', {
+        ...args, parent: window.p4_vars.options.take_action_page,
+      }) || [],
+      ...(window.p4_vars.options.new_ia === 'on' ?
+        (select('core').getEntityRecords('postType', 'p4_action', args) || []) :
+        []
+      )
+    ).filter(a => `${a.id}` !== postId).sort((a, b) => {
```

Comment:
Needs to filter the edited post id

Reply:
I feel like it would be better to compare the ids as numbers, no? Using `parseInt`?

Reply:
Remember that `postId` is already a string. If you prefer to compare betweens you might need to parse both values, `a.id` and `postId` as `integer`s. Do you agree?

Reply:
And that's why I've implemented as template literals

Reply:
I think is less effort to implement template strings, well, it's what is done 😆